### ISSUE TITLE
Outlook error when clicking on date

### ISSUE THREAD

**Issue by Niels838** [8bitPit/Niagara-Issues#1162]
Hi,

I'm very sorry if this is already a known issue but i could not find it.

When i click on the date on the home screen below the clock, i get the option to choose Google Calendar or Outlook (the app i always use). Google Calendar works fine but when i choose Outlook, it won't open the app and shows an error.

 - Device: Motorola G6
 - Android version: 9
 - Niagara 0.13.0-alpha (previous versions also show the error)

**Comment by Niels838** [comment]
I also tried to clear the defaults of Google Calendar and Outlook, restarting/reinstalling the launcher.

**Comment by 8bitPit** [comment]
@Niels838 Thanks for your request. Can you please post a screenshot displaying how the error message looks?

**Comment by Niels838** [comment]
Hi!

After tapping on the date, i get a white flash that disappears immediately. I think the Outlook app is trying to open but closes again. Followed by the error on the bottom of the screen like on the picture (=error in dutch)

![Screenshot_20200202-212837](https://user-images.githubusercontent.com/60567446/73614954-43d3f800-4604-11ea-84c9-db81cb5bddc4.png)

**Comment by Niels838** [comment]
I also made a GIF of the error. That should make things easier to show.

Thanks in advance!

![ezgif-6-4c683ef3895b](https://user-images.githubusercontent.com/60567446/73615310-1177ca00-4607-11ea-8127-cff3d9e4f080.gif)

**Comment by 8bitPit** [comment]
Thank you for the screenshot & gif, might be caused by Outlook, but I can't say for sure. Have you tried replicating this on another launcher?

**Comment by creamofpayne** [comment]
Just wanted to say I am having the same problem. The GIF posted matches my experience.

I'm not sure how to test this on another launcher. The other launchers I've looked at do not have an integrated calendar widget like Niagara. All other launchers have me creating an Outlook widget which works, but would also work in Niagara (just incredibly ugly).

I did successfully create a custom KWGT Widget and enabled it for a Touch action to open the "Calendar Shortcut" option. Here's a GIF of the error I see with Niagara, and then the options I set to make the KWGT widget work:

![GIF-200205_142233](https://user-images.githubusercontent.com/60715228/73884419-0dc08f00-4823-11ea-811b-1dd819b83939.gif)
